<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VODINHHOI" failures="1" tests="4" name="LoginTest" time="0.262" errors="0" timestamp="2025-02-23T23:03:22 ICT">
  <testcase classname="com.poly.service.LoginTest" name="testUserLogin" time="0.166"/>
  <testcase classname="com.poly.service.LoginTest" name="testUserLogin" time="0.007"/>
  <testcase classname="com.poly.service.LoginTest" name="testUserLogin" time="0.011">
    <failure type="java.lang.AssertionError" message="Kết quả kiểm thử không đúng! expected [Tài khoản hoặc mật khẩu không đúng] but found [Tài khoản không tồn tại]">
      <![CDATA[java.lang.AssertionError: Kết quả kiểm thử không đúng! expected [Tài khoản hoặc mật khẩu không đúng] but found [Tài khoản không tồn tại]
at com.poly.service.LoginTest.testUserLogin(LoginTest.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testUserLogin -->
  <testcase classname="com.poly.service.LoginTest" name="testUserLogin" time="0.006"/>
</testsuite> <!-- LoginTest -->
